example/example/models.py

```diff
@@ -58,3 +58,35 @@ class ExampleEpochModel(models.Model):
 #         models.CharField(max_length=255, blank=True, null=True),
 #         null=True, blank=True,
 #     )
+
+
+class ExampleUnloadableModelManager(models.Manager):
+    def get_by_natural_key(self, name):
+        return self.get(name=name)
+
+
+class ExampleUnloadableParentModel(models.Model):
+
+    objects = ExampleUnloadableModelManager()
+
+    name = models.CharField(unique=True, max_length=255)
+
+    def __str__(self):
+        return self.name
+
+    def natural_key(self):
+        return (self.name,)
+
+
+class ExampleUnloadableChildModel(models.Model):
+
+    objects = ExampleUnloadableModelManager()
+
+    name = models.CharField(unique=True, max_length=255)
+    parent = models.ForeignKey(ExampleUnloadableParentModel, on_delete=models.CASCADE, related_name="children")
+
+    def __str__(self):
+        return self.name
+
+    def natural_key(self):
+        return (self.name,)
```

Comment:
What is the purpose of these classes? Being prefixed with `Example`, are they really just examples that users can load and unload to see how the process works?

Reply:
They are models belonging to the `example` app.  This is an _app_ that is part of the *django-astrosat-core* _project_.  It is used to drive tests and migrations and allows me to play w/ the code in the `astrosat` app w/out dealing w/ the complexity of linking it into a separate, complex, project.